checks/checkforcontent.go

```diff
@@ -89,6 +90,16 @@ func getTarReader(in io.Reader) (*tar.Reader, error) {
 	return tr, nil
 }
 
+func readEntireFile(tr *tar.Reader, hdr *tar.Header) (content []byte, err error) {
```

Comment:
tar.Header is not being used. Consider removing.

Reply:
good catch!